packages/backend/package.json

```diff
@@ -79,6 +81,7 @@
         "nodemailer": "^6.7.2",
         "nodemailer-express-handlebars": "^5.0.0",
         "passport": "^0.6.0",
+        "passport-google-oauth20": "^2.0.0",
```

Comment:
import a new library that contains the right spec and types for passport  google auth